docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -2,3 +2,279 @@
 
 Quantum ESPRESSO
 ================
+
+Now that we've covered the basics, let's continue with a quick demo of how AiiDA can make your life easier as a computational scientist.
+
+Importing a structure and inspecting it
+---------------------------------------
+
+Let's download a structure from the `Crystallography Open Database <http://crystallography.net/cod/>`_ and import it into AiiDA.
+
+.. note::
+
+    You can also view the structure online `here <http://crystallography.net/cod/9008565.html>`_.
+
+Download the file and import it with the following two commands:
+
+.. code-block:: console
+
+    $ wget http://crystallography.net/cod/9008565.cif
+    $ verdi data structure import ase 9008565.cif
+
+Remember that each piece of data in AiiDA gets a PK number (a "primary key") that identifies it in your database.
+This is printed out on the screen by the ``verdi data structure import`` command.
+It's a good idea to mark it down, but should you forget, you can always have a look at the structures in the database using:
+
+.. code-block:: console
+
+    $ verdi data structure list
+
+.. important::
+
+    Throughout the tutorial, remember to replace the string ``<PK>`` with the appropriate PK number.
+
+Let us first inspect the node you just created:
+
+.. code-block:: console
+
+    $ verdi node show <PK>
+
+You will get in output some information on the node, including its type (``StructureData``, the AiiDA data type for storing crystal structures), a label and a description (empty for now, can be changed), a creation time (``ctime``) and a last modification time (``mtime``), the PK of the node and its UUID (universally unique identifier).
+
+``StructureData`` can be exported to file in various formats.
+As an example, let's export the structure in XSF format and visualize it with XCrySDen:
+
+.. code-block:: console
+
+    $ verdi data structure export --format=xsf <PK> > exported.xsf
+    $ xcrysden --xsf exported.xsf
```

Comment:
I wasn't able to open xcrysden through the ssh channel (I could open pdfs with evince just fine), did anybody else try this? Maybe we need to be careful when setting the ssh config.

Reply:
I'll try it myself. If it doesn't work, we can always remove it.